packages/engine/src/datastore/arrow/element_conversion.rs

```diff
@@ -278,7 +280,7 @@ pub mod tests {
                 col_element_to_json_val(
                     &array_ref,
                     idx,
-                    &DataType::List(Box::new(DataType::UInt32))
+                    &DataType::List(Box::new(ArrowField::new("item", DataType::UInt32, true))),
```

Comment:
Yes, this was one of the biggest changes for Arrow and is now on par with the C++ implementation (`List<Field>`). I'm currently investigating this because this is probably one reason, why some of the tests are failing.

[Asana task](https://app.asana.com/0/1201793759102797/1201849814189755) (_internal_)